src/main/include/subsystems/Elevator.h

```diff
@@ -24,24 +24,44 @@ namespace t34
   class Elevator : public frc2::SubsystemBase
   {
   public:
+
+  enum class WristType
+  {
+    kAlgae,
+    kCoral
+  };
+
     Elevator();
 
     frc2::CommandPtr ElevateToCommand(units::inch_t height);
-    frc2::CommandPtr MoveWristToCommand(units::degree_t angle);
+    frc2::CommandPtr MoveWristToCommand(WristType wrist, units::degree_t angle);
     frc2::CommandPtr MoveUpOnceCommand();
     frc2::CommandPtr MoveDownOnceCommand();
     frc2::CommandPtr MoveToLevelCommand(int level);
+    frc2::CommandPtr MoveToRestCommand();
+
+    frc2::CommandPtr MoveElevatorByPower(double val);
+    frc2::CommandPtr MoveAlgaeWristByPower(double val);
+    frc2::CommandPtr MoveCoralWristByPower(double val);
 
   private:
     int m_level;
 
-    SparkMax m_wrist_motor;
+    bool m_at_rest;
+
+    units::inch_t m_init_height;
+
+    units::degree_t m_init_coral_angle;
+    units::degree_t m_init_algae_angle;
```

Comment:
The initial height/angles do not appear to change throughout the lifetime of the program. Suggest making them `const`:

```suggestion
    const units::inch_t m_init_height;

    const units::degree_t m_init_coral_angle;
    const units::degree_t m_init_algae_angle;
```